Data/Uniques/body.lua

```diff
@@ -1046,13 +1046,13 @@ Shroud of the Lightless
 Carnal Armour
 League: Abyss
 Source: Drops from Abyssal Liches
-Variant: One Abyssal Socket
 Variant: Two Abyssal Sockets
+Variant: One Abyssal Socket
 Requires Level: 71, 88 Dex, 122 Int
 Implicits: 1
 +(20-25) to maximum Mana 
-{variant:1}Has 1 Abyssal Socket
 {variant:2}Has 2 Abyssal Sockets
+{variant:1}Has 1 Abyssal Socket
```

Comment:
```suggestion
{variant:1}Has 2 Abyssal Sockets
{variant:2}Has 1 Abyssal Socket
```

You need to flip these around, otherwise the text doesn't match the mod

Reply:
Woops. Did not check this thoroughly and since I saw that the variant list updated, I figured the mod was also flipped correctly. Fixed now.